doc/celeste_equations.lyx

```diff
@@ -1630,6 +1630,170 @@ Also,
 
```

Comment:
I think with `git rebase`, you can reorder commits as well as squash related commits into one. If these doc changes are different from the ForwardDiff change, you might squash the ForwardDiff changes together, but leave this one as its own.